pagopa/io-app
# feat: [IOPAE-1316] Display an informative message if the search does not return results

## Short description
This PR adds an informative message if the search does not return results

<details open><summary>Details</summary>
<p>

| search |
| - | 
| <video src="https://github.com/user-attachments/assets/b3b448c6-35c8-4175-833a-6b4c31770d8f" /> |

</p>
</details> 

## List of changes proposed in this pull request
- updated `SearchScreen` to display an information message when the search does not return results

## How to test
Navigate to the search screen and check that the information message is displayed when the search does not return results.
